shared/settings/feedback-container.js

```diff
@@ -49,57 +49,51 @@ class FeedbackContainer extends Component<void, {status: string} & TimerProps, S
         reject(new Error('Not implemented on Desktop!'))
       }
 
-      if (isIOS) {
-        // We don't get the notification from the daemon so we have to do this ourselves
-        const logs = []
-        console.log('Starting log dump')
-
-        const logNames = ['actionLogger', 'storeLogger']
-
-        logNames.forEach(name => {
-          try {
-            const logger = getLogger(name)
-            logger &&
-              logger.dumpAll((...args) => {
-                logs.push(args)
-              })
-          } catch (_) {}
-        })
-
-        logs.push(['=============CONSOLE.LOG START============='])
-        const logger = getLogger('iosConsoleLog')
-        logger &&
-          logger.dumpAll((...args) => {
-            // Skip the extra prefixes that period-logger uses.
-            logs.push([args[1], ...args.slice(2)])
-          })
-        logs.push(['=============CONSOLE.LOG END============='])
+      const logs = []
+      console.log('Starting log dump')
 
-        const path = `${cachesDirectoryPath}/Keybase/rn.log`
-        console.log('Starting log write')
+      const logNames = ['actionLogger', 'storeLogger']
 
-        writeStream(path, 'utf8')
-          .then(stream => {
-            const writeLogsPromises = logs.map((log, idx) => {
-              return () => {
-                return stream.write(JSON.stringify(log, null, 2))
-              }
+      logNames.forEach(name => {
+        try {
+          const logger = getLogger(name)
+          logger &&
+            logger.dumpAll((...args) => {
+              logs.push(args)
             })
+        } catch (_) {}
+      })
+
+      logs.push(['=============CONSOLE.LOG START============='])
+      const logger = getLogger('consoleLog')
+      logger &&
+        logger.dumpAll((...args) => {
+          // Skip the extra prefixes that period-logger uses.
+          logs.push([args[1], ...args.slice(2)])
+        })
+      logs.push(['=============CONSOLE.LOG END============='])
 
-            return serialPromises(writeLogsPromises).then(() => stream.close())
-          })
-          .then(success => {
-            console.log('Log write done')
-            resolve(true)
-          })
-          .catch(err => {
-            resolve(false)
-            console.warn(`Couldn't log send! ${err}`)
+      const path = isIOS ? `${cachesDirectoryPath}/Keybase/rn.log` : `${filesDirectoryPath}/rn.log`
```

Comment:
keeping the paths different for now, reduce how much code i need to change to make this work